### ISSUE TITLE
Mobile Access to Bookmarklet

### ISSUE THREAD

**Issue by mrjarbenne** [PressForward/pressforward#935]
The core "Press This" functionality in WP adds a few options to make it easier to get the bookmarklet javascript into the bookmarks bar. It would be great if Pressforward offered similar functionality for uses to be able to add the bookmarklet on mobile devices.

![2017-08-20_13-57-01](https://user-images.githubusercontent.com/480667/29497084-cca763e6-85af-11e7-9609-efa505593d7e.png)

**Comment by regan008** [comment]
@AramZS is this possible in the bookmarklet redesign?

**Comment by AramZS** [comment]
Yes, but also we can do this now with the bookmarklet, I think we have instructions somewhere

**Comment by mrjarbenne** [comment]
The only way I have been able to do this is to create the bookmarklet on a laptop, and then email the javascript to myself so that I can copy it and then paste it into an existing bookmark in iOS Safari, replacing the URL from the existing bookmark with the Javascript, and renaming it "Press Forward". As you can see in the video below, pressing the Open button does nothing, and attempting to open the button in a new tab kicks out a javascript error. 

![2017-09-11_23-55-56](https://user-images.githubusercontent.com/480667/30307600-40260cba-974d-11e7-9f93-7ae725a68a06.gif)

I would love a way to handle this when supporting teachers on iPads if you have a work around. 

Thanks again.

**Comment by boonebgorges** [comment]
It's hard to make this change without first implementing some other changes from Press This. See #448.

**Comment by boonebgorges** [comment]
Because of the way that Nominate This works in PF, it's not possible to have the 'Direct Link' button from Press This.

I am, however, able to add the 'clipboard' button. I've done so, and cleaned up the language on this panel a bit. The "clipboard closed" and "clipboard open" views are below:

![Screenshot_2023-05-11_09-25-26](https://github.com/PressForward/pressforward/assets/246627/86e5f5f3-239d-438a-9097-609001ff7b60)

![Screenshot_2023-05-11_09-26-01](https://github.com/PressForward/pressforward/assets/246627/1b02e7d9-4119-4ad4-a8b1-161f8dd9fdec)

**Comment by AramZS** [comment]
I'll add that this is exactly the way I use the bookmarklet on mobile. Copying it out of this field and pasting it into a bookmark's URL field lets me use Nominate This pretty effectively on mobile!
